src/app/components/AddressBox/index.tsx

```diff
@@ -7,18 +7,23 @@ import copy from 'copy-to-clipboard'
 import { Box, Button, Text } from 'grommet'
 import { Copy } from 'grommet-icons/icons'
 import React, { memo } from 'react'
+import { useTranslation } from 'react-i18next'
 
 import { PrettyAddress } from '../PrettyAddress'
+import { useToastNotification } from '../Notification'
 
 interface Props {
   address: string
 }
 
 export const AddressBox = memo((props: Props) => {
+  const { t } = useTranslation()
+  const { showNotification } = useToastNotification()
   const address = props.address
 
   const copyAddress = () => {
     copy(address)
+    showNotification(t('account.addressCopied', 'Address copied.'))
```

Comment:
Can you add a test to cover this?

Reply:
I have now added a test suite for the `AddressBox` component, which covers both the clipboard and test notification.